ui/components/multichain/network-list-menu/new-network-list-menu/network-list-menu.tsx

```diff
@@ -0,0 +1,814 @@
+import React, { useContext, useEffect, useState } from 'react';
```

Comment:
We don't need a new menu file; additional headers, etc. should be in `ui/components/multichain/network-list-menu/network-list-menu.js`